lib/y2logsstep.pm

```diff
@@ -263,6 +263,8 @@ sub save_system_logs {
         assert_script_run 'btrfs filesystem usage /mnt | tee /tmp/btrfs-filesystem-usage-mnt.txt';
         upload_logs '/tmp/btrfs-filesystem-df-mnt.txt';
         upload_logs '/tmp/btrfs-filesystem-usage-mnt.txt';
+        # Detect bsc#1063638
+        record_soft_failure 'bsc#1063638' if (script_run('ps x | grep "btrfs-\(scrub\|balance\|trim\)"') == 0);
```

Comment:
as discussed in person best to move to opensusebasetest

Reply:
Sounded like a good idea, now I doubt that, as we don't open any kind of console in `opensusebasetest` postfail_hook, and `problem_detection` is not called from everywhere. I will check where we can copy it, as moving doesn't help.